.eslintrc

```diff
@@ -73,7 +73,8 @@
         "@typescript-eslint/no-var-requires": "off",
         "@typescript-eslint/no-empty-interface": "off",
         "@typescript-eslint/naming-convention": [
-            "error",
+            "off",
+            // changed from error to off for consistency for deprecating to "v2_"
```

Comment:
instead of a blanket ignore rule, can we only restrict this to `v2_`? we should enforce this convention for all other non-deprecated cases

can take a look at the `filter` option from the documentation https://typescript-eslint.io/rules/naming-convention/#filter

Reply:
as discussed, no longer required after switching to `V2_`